[2025-08-18 11:15:43,732] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: send_weather_to_kafka.verify_pipeline scheduled__2025-08-18T09:15:37.816351+00:00 [queued]>
[2025-08-18 11:15:43,739] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: send_weather_to_kafka.verify_pipeline scheduled__2025-08-18T09:15:37.816351+00:00 [queued]>
[2025-08-18 11:15:43,739] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2025-08-18 11:15:43,739] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2025-08-18 11:15:43,739] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2025-08-18 11:15:43,750] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): verify_pipeline> on 2025-08-18 09:15:37.816351+00:00
[2025-08-18 11:15:43,753] {standard_task_runner.py:52} INFO - Started process 2471 to run task
[2025-08-18 11:15:43,756] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'send_weather_to_kafka', 'verify_pipeline', 'scheduled__2025-08-18T09:15:37.816351+00:00', '--job-id', '278', '--raw', '--subdir', 'DAGS_FOLDER/dag_meteo.py', '--cfg-path', '/tmp/tmp3rub0dpo', '--error-file', '/tmp/tmp2dil6b8q']
[2025-08-18 11:15:43,757] {standard_task_runner.py:80} INFO - Job 278: Subtask verify_pipeline
[2025-08-18 11:15:43,784] {logging_mixin.py:115} WARNING - /home/***/.local/lib/python3.7/site-packages/***/configuration.py:470 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-08-18 11:15:43,836] {task_command.py:369} INFO - Running <TaskInstance: send_weather_to_kafka.verify_pipeline scheduled__2025-08-18T09:15:37.816351+00:00 [running]> on host a4b544519f2b
[2025-08-18 11:15:43,929] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=send_weather_to_kafka
AIRFLOW_CTX_TASK_ID=verify_pipeline
AIRFLOW_CTX_EXECUTION_DATE=2025-08-18T09:15:37.816351+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2025-08-18T09:15:37.816351+00:00
[2025-08-18 11:15:43,930] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2025-08-18 11:15:43,930] {subprocess.py:74} INFO - Running command: ['bash', '-c', '\n            echo " Attente traitement Logstash..." && sleep 30 &&\n            curl -s "http://elasticsearch:9200/_cat/indices/weather*" || echo "Pas encore d\'index weather"\n        ']
[2025-08-18 11:15:43,943] {subprocess.py:85} INFO - Output:
[2025-08-18 11:15:43,944] {subprocess.py:92} INFO -  Attente traitement Logstash...
[2025-08-18 11:16:13,985] {subprocess.py:92} INFO - yellow open weather_data BQjbDtS5TvO1Q2HrAJGyvA 1 1 68 0 35.3kb 35.3kb
[2025-08-18 11:16:13,988] {subprocess.py:96} INFO - Command exited with return code 0
[2025-08-18 11:16:14,013] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=send_weather_to_kafka, task_id=verify_pipeline, execution_date=20250818T091537, start_date=20250818T111543, end_date=20250818T111614
[2025-08-18 11:16:14,042] {local_task_job.py:156} INFO - Task exited with return code 0
[2025-08-18 11:16:14,090] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
